src/main/resources/application.properties

```
@@ -1 +1 @@
-
+spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
```

```suggestion
spring:
    mvc:
        pathmatch:
            matching-strategy: ant_path_matcher
```
이런 구성인 `application.yml` 형식으로 바꾸는건 어떨까요? yaml이 계층으로 표현해서 가독성이 더 좋을 것 같아 제안드려요😀

---

아 그게 좋을것같네요! 바꾸겠습니다!